init/eessi_environment_variables

```diff
@@ -23,6 +23,11 @@ if [ -d $EESSI_PREFIX ]; then
 
   # Set EPREFIX since that is basically a standard in Gentoo Prefix
   export EPREFIX=$EESSI_PREFIX/compat/$EESSI_OS_TYPE/$EESSI_CPU_FAMILY
+
+  # add location of commands provided by compat layer to $PATH;
+  # see https://github.com/EESSI/software-layer/issues/52
+  export PATH=$EPREFIX/usr/bin:$PATH
```

Comment:
Just to verify: do we only need `$EPREFIX/usr/bin`? The `startprefix` script also adds `/(s)bin` and `/usr/sbin` (and  `/usr/local/(s)bin` and `/opt/bin`, which we don't have in our installation).

Reply:
I would start with only `$EPREFIX/usr/bin`, since that's where all the tools are that we've been having trouble with (`ld`, `curl`, `python`).

Long-term, I'm not convinced this is the best way forward, but as suggested in #52 it makes sense to try this approach and see if we hit any trouble with it.

Reply:
Okay!